.github/workflows/static.yaml

```
@@ -1,50 +1,55 @@
-# 静的コンテンツを GitHub Pages にデプロイするためのシンプルなワークフロー
 name: Deploy static content to Pages
 
 on:
-  # デフォルトブランチを対象としたプッシュ時にで実行されます
   push:
-    branches: ["develop"]
-
-  # Actions タブから手動でワークフローを実行できるようにします
+    branches: ["feature/*"]
```

```suggestion
    branches: ["feature/*", "bugfix/*"]
```
ついでにbugfixも足しといてください！

---

追加しました